### DISCUSSION TITLE
Flunt - o que posso usar no lugar do .HasLen()

### DISCUSSION THREAD

**Discussion by under-sin** [balta-io/forum#218] [Tira Dúvidas] [answered]
Pessoal, estou fazendo o curso de **Refatorando para testes de unidade** e aqui o Balta usa o **Flunt** em uma versão antiga.
Nessa versão ele usar muito o 
```
.HasLen(object, int, key, menssage)
```
Porém, na versão que instalei (2.0.5) não tem esse método. Alguém sabe se tem um método equivalente ?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @under-sin, como vai?
Recomendo que primeiro faça o curso com a mesma versão do Flunt (1.0.5 se não me engano).

Após isto, se quiser utilizar a 2.0, o HasLen mudou para `IsGreaterThan` ou `IsLowerThan` (Ou ambos).
👉 https://github.com/andrebaltieri/Flunt/blob/main/Flunt/Validations/StringValidationContract.cs
